a2rchi/templates/base-compose.yaml

```diff
@@ -59,34 +58,30 @@ services:
       - ./config.yaml:/root/A2rchi/config.yaml
       - ./main.prompt:/root/A2rchi/main.prompt
       - ./condense.prompt:/root/A2rchi/condense.prompt
-      - ./summary.prompt:/root/A2rchi/summary.prompt
+      {% if gpu_ids -%}
+      - a2rchi-models:/root/models/
```

Comment:
I saw this hardcoded in models.py ... if it's the same, can we read it from here?